packages/analytics-js/src/services/StoreManager/Store.ts

```diff
@@ -38,7 +38,7 @@ class Store implements IStore {
     this.name = config.name;
     this.isEncrypted = config.isEncrypted ?? false;
     this.validKeys = config.validKeys ?? {};
-    this.engine = engine ?? getStorageEngine(LOCAL_STORAGE);
+    this.engine = engine;
```

Comment:
This is an internal class and `engine` is a required parameter. So, having a default value does not make sense.